src/builder/create_message.rs

```diff
@@ -40,16 +41,16 @@ use utils::{self, VecMap};
 /// [`embed`]: #method.embed
 /// [`http::send_message`]: ../http/fn.send_message.html
 #[derive(Clone, Debug)]
-pub struct CreateMessage(pub VecMap<&'static str, Value>, pub Option<Vec<ReactionType>>);
+pub struct CreateMessage<'a>(pub VecMap<&'static str, Value>, pub Option<Vec<ReactionType>>, pub Vec<AttachmentType<'a>>);
```

Comment:
Due to the introduction of this lifetime and a new field, the target branch needs to be changed to v0.6.x.